tools/generate_from_protos.sh

```diff
@@ -24,3 +25,23 @@ for plugin in ${plugin_list}; do
   mkdir -p ${gdir}
   ${protoc_binary} -I ${proto_dir} --rust_out=${gdir} --grpc_out=${gdir} --plugin=protoc-gen-grpc=`which grpc_rust_plugin` ${proto_dir}/${plugin}/${plugin}.proto
 done
+
+echo ""
+echo "-------------------------------"
+echo "Generating the SDK wrappers"
+echo "-------------------------------"
+echo ""
+
+if [ ! -d ${pb_plugins_dir}/venv ]; then
+    python3 -m venv ${pb_plugins_dir}/venv
+
+    source ${pb_plugins_dir}/venv/bin/activate
+    pip install -r ${pb_plugins_dir}/requirements.txt
+    pip install -e ${pb_plugins_dir}
+fi
+
+source ${pb_plugins_dir}/venv/bin/activate
+
+for plugin in ${plugin_list}; do
+    ${protoc_binary} -I ${proto_dir} --plugin=protoc-gen-custom=`which protoc-gen-dcsdk` --custom_out=${backend_generated_dir} --custom_opt=file_ext=rs,template_path=${script_dir}/../templates ${proto_dir}/${plugin}/${plugin}.proto
```

Comment:
This is the line running our protoc plugin, called `protoc-gen-dcsdk`. This is the one taking the templates and generating the output file "<Plugin>.rs"